html-forms/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+    <p><label>Email:</label></p>
+    <input type="text" name="email">
+    <p><label>Why is coding cool?</label></p>
+    <select name="Why is coding cool">
+    <option value="Because it's fun"><label for="Because it's fun">Because it's fun</label></option>
+    <option value="Because it's interesting"><label for= "Beacause it's interesting">Because it's interesting</label></option>
+    <option value="Because it's powerful"><label for="Because it's powerful">Because it's powerful</label></option>
```

Comment:
`option` elements can't have `label`s, you can just have the text content inside the `option` element.